### ISSUE TITLE
v2: Duplicate access log with Caddyfile

### ISSUE THREAD

**Issue by carlidev** [caddyserver/caddy#3110]
With the following configuration and caddy b15, the access traces in the logs are duplicated :
```
localhost:80 {
    respond "Hello, world!"
    log {
        output stdout
    }
}

app.localhost:80 {
    respond "Hello, world!"
    log {
        output stdout
    }
}
```

When I execute `curl localhost` (only once), i get the following three entries in the logs :
```
2020/03/02 12:51:04.902 [34mINFO [0m   http.log.access.log0...
2020/03/02 12:51:04.902 [34mINFO [0m   http.log.access.log0...
2020/03/02 12:51:04.902 [34mINFO [0m   http.log.access.log0...
```
There should probably only be one trace.

**Comment by tobya** [comment]
Thank you for the issue.

 - Can you provide full logs, these look like they have been truncated.
 - What happens if you set Output to a filename?

**Comment by carlidev** [comment]
Thanks for responding,

Below is the original test with commands and full traces:
![issue-3110-stdout](https://user-images.githubusercontent.com/36325732/76142300-86a64700-606c-11ea-9eb9-e2cb090f61e8.png)

Below is the test with Ouput to filenames (with commands and full traces):
![issue-3110-file](https://user-images.githubusercontent.com/36325732/76142489-a2aae800-606e-11ea-9436-1e42584a9fe7.png)

Let's hope that's clear,
Best regards

**Comment by mholt** [comment]
Alrighty, fixed this in 6b60a30. You will still see a second log entry in your case, because both your default log (which is implicit) and your access loggers go to the terminal, but now you will only see two instead of three.
